resource/command.go

```diff
@@ -12,15 +12,15 @@ import (
 )
 
 type Command struct {
-	Title      string   `json:"title,omitempty" yaml:"title,omitempty"`
-	Meta       meta     `json:"meta,omitempty" yaml:"meta,omitempty"`
-	Command    string   `json:"-" yaml:"-"`
-	Exec       string   `json:"exec,omitempty" yaml:"exec,omitempty"`
-	ExitStatus matcher  `json:"exit-status" yaml:"exit-status"`
-	Stdout     []string `json:"stdout" yaml:"stdout"`
-	Stderr     []string `json:"stderr" yaml:"stderr"`
-	Timeout    int      `json:"timeout" yaml:"timeout"`
-	Skip       bool     `json:"skip,omitempty" yaml:"skip,omitempty"`
+	Title      string  `json:"title,omitempty" yaml:"title,omitempty"`
+	Meta       meta    `json:"meta,omitempty" yaml:"meta,omitempty"`
+	id         string  `json:"-" yaml:"-"`
```

Comment:
no harm, so just a comment - the `json` and `yaml` tags here not needed as the marshallers wont include private struct members